bootstrap.sh

```diff
@@ -1,27 +1,27 @@
 #!/usr/bin/env bash
 
-cd "$(dirname "${BASH_SOURCE}")";
+cd "$(dirname "${BASH_SOURCE}")"
 
-git pull origin master;
+git pull origin master
 
-function doIt() {
+doIt() {
 	rsync --exclude ".git/" \
 		--exclude ".DS_Store" \
 		--exclude ".osx" \
 		--exclude "bootstrap.sh" \
 		--exclude "README.md" \
 		--exclude "LICENSE-MIT.txt" \
-		-avh --no-perms . ~;
-	source ~/.bash_profile;
+		-avh --no-perms . ~
+	source ~/.bash_profile
 }
 
-if [ "$1" == "--force" -o "$1" == "-f" ]; then
-	doIt;
+if [[ $1 == --force || $1 == -f ]]; then
```

Comment:
Sure! If you take a look at `man [`, the documentation of the `[` command, it explains the `=` operator, but the `==` operator is nowhere to be found. It's a mystery it works at all.

The `==` operator is explained for the more modern `[[` command in `help [[`.

And the `-o` operator is not recommended.

See also the comments on [this question](http://stackoverflow.com/q/38682618/641955) and [its answer](http://stackoverflow.com/a/38684324/641955) on Stack Overflow.